tests/XSPM/Test-XspmDevices.Tests.ps1

```diff
@@ -0,0 +1,29 @@
+﻿BeforeDiscovery {
+    try {
+        $DefenderPlan = Get-MtLicenseInformation -Product "DefenderXDR"
+    } catch {
+        $DefenderPlan = "NotConnected"
```

Comment:
Can this catch block be updated to accurately reflect whether the session is not connected _or_ does not have the required licenses?